samples/net/sockets/echo_async_select/src/socket_echo_select.c

```diff
@@ -0,0 +1,214 @@
+/*
+ * Copyright (c) 2017 Linaro Limited
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <stdio.h>
+#include <stdbool.h>
+#include <errno.h>
+#include <stdlib.h>
+
+#ifndef __ZEPHYR__
+
+#include <netinet/in.h>
+#include <sys/socket.h>
+#include <arpa/inet.h>
+#include <unistd.h>
+#include <fcntl.h>
+
+#else
+
+#include <logging/log.h>
```

Comment:
I'm pretty sure you no longer need to create bougs loggger instance if you don't use it.

Reply:
> I'm pretty sure you no longer need to create bougs loggger instance if you don't use it.

Ok, removed that. Note that otherwise this sample is a clone of echo_async, and it would be nice to keep them in sync, so people could diff them and see the real differences in API use. So, more substantial changes would be need to done to both samples, which is hopefully outside the scope of this PR.